setup/configuration/index.md

```diff
@@ -160,6 +190,9 @@ ci:
 
 The Kubernetes, Google Cloud, and App Engine cloud providers can load account information from files that are separate from the service YAML configuration. You can load these separate files from an external source using the Config Server's Resource abstraction. In the service YAML configuration, prefix a file path with `configserver:` to indicate that the file should be retrieved by the Config Server. 
```

Comment:
There's no Google Cloud provider, per se. There's Compute Engine, App Engine (which is listed here), and Kubernetes Engine. I assume this is a reference to Compute Engine?

Reply:
Fixed. Thanks!